app/format/format.tsx

```diff
@@ -116,6 +116,15 @@ export function formatCommitHash(commit: string) {
   return commit?.substring(0, 6);
 }
 
+export function formatWithCommas(num: Long) {
+  var l = String(num);
```

Comment:
nit: use `const` whenever possible (instead of `var`) -- https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75